src/views/properties.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import BootstrapTable from 'react-bootstrap-table-next';
+import { UserContext } from '../App';
+import { Link } from "react-router-dom"
+import * as axios from 'axios';
+
+const columns = [{
+    dataField: 'name',
+    text: 'Name',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'propertyManager',
+    text: 'Property Managers',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'address',
+    text: 'Address',
+    sort: true,
+    headerStyle: () => {
+      return { width: "20%" };
+    }
+  }, {
+    dataField: 'tenants',
+    text: 'Tenants',
+    sort: true,
+    headerStyle: () => {
+      return { width: "10%" };
+    }
+  }, {
+    dataField: 'dateAdded',
+    text: 'Added On',
+    sort: true,
+    headerStyle: () => {
+      return { width: "10%" };
+    }
+  }];
+
+const selectRow = {
+  mode: 'checkbox',
+  clickToSelect: true,
+  sort: true,
+  headerColumnStyle: () => {
+    return { width: "5%" };
+  }
+};
+
+
+export class Properties extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+        properties: [],
+        }
+
+        this.getProperties = this.getProperties.bind(this)
+    }    
+
+    componentDidMount() {
+        this.getProperties(this.context);
+    }
+
+    getProperties = (context) => {
+        axios.get('http://localhost:5000/properties', { headers: {"Authorization" : `Bearer ${context.user.accessJwt}`} })
```

Comment:
We will need to use the environment variable now instead for URL's:
`${process.env.REACT_APP_API_URL}/properties`

This will allow Heroku to replace the environment variable value with the prod URL